Report.md

```diff
@@ -73,6 +73,28 @@ Design Thinking Approach
 ========================
 *3. Describe your approach from a Design Thinking principles perspective. Your Design Thinking approach to address the problem with artefacts*
 
+## **Approach - Design Thinking**
+Design thinking is a non-linear, iterative process which seeks the understanding of users to challenge assumptions, redefine problems and create innovative solutions to prototype and test. This customer-centric approach was undertaken to address the improvements requested by the Major Travel Company for their Call Management Centre (CMC). The design thinking approach undertook consists of the execution of five phases, (1) empathize, (2) define, (3) ideate, (4) prototype and (5) test and by addressing these phrases we derived customer-centric innovation. 
+
+### Empathise
+Phrase one consisted of empathising with stakeholders, this represents anyone who is impacted by the project being developed with a particular focus extended on the end customer. Empathy is a crucial step of achieving a human-centred design process and was utilised to gain insight into users and their needs without introducing any pre imposed bias. When we create solutions, we get better solutions when we think of those who they will affect. To document these perspectives the tool of an empathy map was utilised to collaboratively visualise and share our understanding of stakeholders needs and aid future decision making. This stage allowed us to gain a deeper understanding of the perspective of the users involved and how to shape our solution. 
+
+### Define
+Phrase two of defining allowed us to synthesis the knowledge obtained in the empathise stage and define the core problems to solve. Here the problem statements were directly derived from creating point of view (POV) statements for all the stakeholders identified. This staged allowed us as a team to consolidate assumptions and agree on features to be developed to solve the problem given. 
+
+### Ideate
+Phrase three consisted of utilising the user research conducted to ideate solutions to the problem statements we created in the prior stage. The brainstorming and worst possible idea techniques were utilised to assist within our ideation stage. These techniques were chosen to initiate freeform group discussions and simulated creative solutions in light of the problems identified. These sessions consisted of developing “how might we? (HMW)” statements and deriving features relative to the point of view statements developed in the define stage. The conclusions of the brainstormed were documented in (need to do this???). At the conclusion of this stage, the agreed features were converted to issues to be developed for the prototype and testing stage. 
```

Comment:
Yeah, I'm pretty sure we'll need to document them in. I was going to do it but confused myself so I will ask what he wants from the 200-300 word reflection before attempting it.

Reply:
Yea I was also confused when I saw that session - since we did do brainstorming but the reflection is something we can to in today's lesson :)